themes/digital.gov/src/scss/new/_promos.scss

```diff
@@ -47,62 +47,71 @@
     @include u-padding-y("2px");
     @include u-radius("pill");
   }
+
   img {
     @include u-margin-top(0);
     @include u-width(3);
   }
+
   img.thumb {
     @include u-margin-top("05");
     @include u-margin-right(2);
     @include u-width(5);
     @include u-height(5);
   }
+
   p {
     @include u-margin-top(0);
-    // @include u-font('sans', 'md');
-    // @include u-line-height('sans', 3);
-  }
-  a {
-    @include u-text("indigo-80");
-    @include u-text("no-underline");
   }
 
-  // To make the whole PROMO clickable, we put and additional <a> tag with a <span> tag in the markup that stretches to the full height and width of the element.
-  position: relative;
   &:hover {
     @include u-bg("gray-cool-5");
+
     a {
       @include u-border-bottom("05", "accent-warm", "solid");
     }
+
     p a {
       @include u-border-bottom("05", "accent-warm", "solid");
     }
   }
+  @include u-margin-y(1);
+  @include u-padding(1);
+  @include u-border("1px", "gray-cool-10", "solid");
+  @include u-radius("md");
+  @include u-display("flex");
+  @include u-flex("align-center");
+  @include u-flex("justify");
```

Comment:
Can we move these to the top where they belong? Keeping them here makes it hard to tell what exactly it's styling.